cli/user/src/test/java/io/pravega/cli/user/kvs/KVTCommandsTest.java

```diff
@@ -22,7 +23,8 @@
 public class KVTCommandsTest extends AbstractUserCommandTest {
 
     @Test(timeout = 10000)
-    public void testCreateKVT() throws Exception {
+    @SneakyThrows
+    public void testCreateKVT() {
```

Comment:
nit: Please avoid abbreviations to the extent possible. `testCreateKeyValueTable()` sounds better. Same for other tests below.

Reply:
done